src/bin/scheduler/setup/windows_permissions.rs

```diff
@@ -30,19 +30,21 @@ pub fn grant_permissions_to_all_plan_users(
     path: &Utf8Path,
     plans: Vec<Plan>,
     permissions: &str,
-    additional_icacls_args: &[&str],
     description_for_failure_reporting: &str,
 ) -> (Vec<Plan>, Vec<SetupFailure>) {
     let mut surviving_plans = vec![];
     let mut failures = vec![];
 
     for (session, plans_in_session) in plans_by_sessions(plans) {
         if let Session::User(user_session) = session {
-            let icacls_permission_arg = format!("{}:{}", user_session.user_name, permissions);
-            let mut icacls_args = vec![path.as_str(), "/grant", &icacls_permission_arg];
-            icacls_args.extend(additional_icacls_args);
-
-            match run_icacls_command(icacls_args).context(format!(
+            match run_icacls_command([
+                path.as_str(),
+                "/grant",
+                &format!("{}:{}", user_session.user_name, permissions),
+                "/T",
+                "/L",
```

Comment:
Are we sure this works with recursion as expected? Maybe a test would be appropriate here.
```
Performs the operation on a symbolic link instead of its destination.
```